### DISCUSSION TITLE
HACS Install

### DISCUSSION THREAD

**Discussion by smish66** [Aleks130699/ha-fpp#14] [Q&A]
Hi, I notice in the new HACS 2.0 integration in Home Assistant that the search does not find the ha-fpp repository to install via HACS. Is the only way to install this now through manual method only?

**Comment by zackbcom** [comment]
> Hi, I notice in the new HACS 2.0 integration in Home Assistant that the search does not find the ha-fpp repository to install via HACS. Is the only way to install this now through manual method only?

I had to install the repository under custom repositories in HACS. 

![image](https://github.com/user-attachments/assets/48699b8f-f16a-476a-9908-c46590eba677)

**Reply by smish66** [reply]
Ok thanks. That worked.
